master.ron

```diff
@@ -2,6 +2,7 @@ Config(
     id: "node-master",
     port: 1234,
     host: "127.0.0.1",
+    metrics_namespace: "master_ron",
```

Comment:
`master` ? ron is the file format, I don't think it's relevant for metrics :)

(same for slave)